scripts/plot_operator_breakdown.py

```diff
@@ -117,7 +118,7 @@
         bar.set_linewidth(0)
 
 # Set labels
-ax.set_yticklabels(["{:,.0%}".format(x) for x in ax.get_yticks()])
```

Comment:
ax.set_yticklabels(["{:,.0%}".format(x) for x in ax.get_yticks()])

Reply:
?

Reply:
Copy/paste fail ...

I wanted to ask for a screenshot as the removed line looks pretty good to me (it's percent and that's what the formatting does, right?).

Reply:
Does the same thing, but if the yticks ever change (which they don't), the formatting would be broken. That's why we were getting a warning there. Also, using the PercentFormatter is just the cleaner, canonical way of doing this.